forge-gui/res/cardsfolder/s/spire_phantasm.txt

```diff
@@ -3,8 +3,8 @@ ManaCost:2 U U
 Types:Creature Gargoyle Illusion
 PT:3/2
 Draft:Reveal CARDNAME as you draft it.
-Draft:The next time a player drafts a card from this booster pack, guess that card’s name. Then that player reveals the drafted card.
+Draft:The next time a player drafts a card from this booster pack, guess that card's name. Then that player reveals the drafted card.
```

Comment:
this is also linked to code I believe

Reply:
Duly noted and will check. 

Also, my bad on not reading [Stone Giant](https://scryfall.com/card/ddi/55/stone-giant) a bit more thoroughly.

Reply:
Done. I hope the mishap working on the version of `LimitedPlayer.java` I had on hard drive without thinking to check if it had been updated on the repository isn't an issue. Also, I notice on my hard drive fork, which I obtain by the "download a *.zip file" option Github provides, there's a `forge-gui\target\classes\forge\gamemodes\limited\LimitedPlayer.class` folder I'm not finding on the repository.